### DISCUSSION TITLE
v1.8.6 (1402-4-16)

### DISCUSSION THREAD

**Discussion by GFW-knocker** [GFW-knocker/MahsaNG#5] [Announcements]
ابتدا نسخه قبلی را حذف و سپس اقدام به نصب نسخه جدید کنید وگرنه به باگ های عجیب برمیخورید

برای گوشی های جدید از نسخه
v8a
و برای گوشی های قدیمی تر از 
v7a
و برای گوشی ویندوزی از
x86
استفاده کنید

اگر به برنامه اعتماد ندارید آن را در سایت
virustotal.com
با تمامی آنتی ویروس ها چک کنید

fix UI hang
fix per-app proxy mode
add clickable support link for every config
add third-party config from telegram
fix reality connecting issue
fix DNS leak (almost)
add fragment option in main window
add initial help dialog

<hr /><em>This discussion was created from the release <a href='https://github.com/GFW-knocker/MahsaNG/releases/tag/v1.8.6-(1402-4-16)'>v1.8.6 (1402-4-16)</a>.</em>

**Comment by fear009** [comment]
اصل هدفتون به کنار که کار قشنگیه ولی فک نمیکنید مردان در این انقلاب خون دادن کشته شدن اعدام شدن و حقشونه اسمی ازشون برده شه؟؟ #انقلاب بدون مردان حکم عروسک بازی داره

**Reply by GFW-knocker** [reply]
تعداد جانباختگان راه آزادی بسیار زیاده مجبوریم مهسا رو به نمایندگی از همه برگزینیم وگرنه تفاوتی بین زن و مرد نداریم
